govgen-1/gentx/gentx-interchainfm.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"interchain.FMsteak","identity":"8E92184569CD8E2D","website":"https://interchain.fm","security_contact":"staking@rockx.com","details":"Non-custodial high 99.99% uptime validator w/ 100% return for downtime slashing. Run by your favorite Cosmos Radio Station."},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"govgen1wcvz87xp998qqfc5zvwja6t0p90eqqvcar0x5z","validator_address":"govgenvaloper1wcvz87xp998qqfc5zvwja6t0p90eqqvcvpx87c","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"grEwHAKG7uZvJ/H+MFbF6dz7cnKeE8GricxTyL9Ri7I="},"value":{"denom":"ugovgen","amount":"1000000"}}],"memo":"aebe4bf557b694414a0b9419f6914989d8b20697@141.94.209.185:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A4bzWNGXGpqNUCHG4OyEa2CpekIyWz+iBHnBdXKUz97g"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["EMkykrXyvCliqKmB0K0rmUCLLCZu60o9AhtHNgDXlZcFumMC/LcrFpr9xJeOUZX5TlrC7+FONAGOA40nMdj8aw=="]}
```

Comment:
please set rate correctly

Reply:
@Chjango all the commission rate fields should be 0.